docs/projects/tad/reporting-standard/latest.md

```diff
@@ -441,6 +465,7 @@ JSON schema will be added when we publish the first beta version.
 
 ## Changelog
 
+- 0.1a5: adds a general description of high-risk systems for the EU technical documentation.
```

Comment:
My proposal is to refer to the EU AI Act i.s.o. EU technical documentation. What do you think?

```suggestion
- 0.1a5: adds a general description of the technical documentation required for high-risk systems to conform to the EU AI Act.
```